src/domain/data-decoder/entities/schemas/data-decoded.schema.ts

```diff
@@ -1,7 +1,10 @@
 import { Schema } from 'ajv';
 
+export const DATA_DECODED_PARAMTER_SCHEMA_ID =
+  'https://safe-client.safe.global/schemas/data-decoded/data-decoded-parameter.json';
+
 export const dataDecodedParameterSchema: Schema = {
-  $id: 'https://safe-client.safe.global/schemas/data-decoded/data-decoded-parameter.json',
+  $id: DATA_DECODED_PARAMTER_SCHEMA_ID,
```

Comment:
I think there's a typo here: `DATA_DECODED_PARAMTER_SCHEMA_ID`, it should be `DATA_DECODED_PARAMETER_SCHEMA_ID` 🙂

Reply:
Good catch! I've fixed it in cb1f6e291adc829fa1db64e44ba640a3573b2e18.